pkg/clusterversion/cockroach_versions.go

```diff
@@ -404,6 +404,12 @@ const (
 	// V23_1KeyVisualizerTablesAndJobs adds the system tables that support the key visualizer.
 	V23_1KeyVisualizerTablesAndJobs
 
+	// V23_1UseEncodingWithBelowRaftAdmissionData enables the use of raft
+	// command encodings that include below-raft admission control data.
+	//
+	// TODO(irfansharif): Actually use this.
```

Comment:
Sorry if I've missed your response to this point on the other PR, but what is the purpose of introducing this before the code that it's gating is there? Introducing a version also implies that the current binary supports the functionality gated behind the version, so this is not the same as checking in unused code.

Reply:
I'll get rid of it.